packages/davinci-client/tsconfig.lib.json

```diff
@@ -13,7 +13,7 @@
     "noImplicitOverride": true,
     "declaration": true,
     "declarationMap": true,
-    "skipLibCheck": true,
+    "skipLibCheck": false,
```

Comment:
i'm unsure about the importance of this change, its probably worth us having it _anyways_ because if theres a bug in an RTK version we are using we probably want to catch it here in our code before publishing.

this setting just checks the library types also